examples/Page.re

```diff
@@ -17,17 +17,17 @@ let make = (~locale, ~setLocale) => {
       </button>
     </div>
     <div className="message">
-      <ReactIntl.DefinedMessage message=pageLocale##hello />
+      <ReactIntl.FormattedMessage id="page.hello" defaultMessage="Hello" />
       " "->React.string
-      <ReactIntl.DefinedMessage message=pageLocale##world />
+      <ReactIntl.FormattedMessage id="page.world" defaultMessage="World" />
     </div>
     <div>
-      <ReactIntl.DefinedMessage message=pageLocale##today />
+      ReactIntl.(intl->Intl.formatMessage(pageLocale##today)->React.string)
```

Comment:
I'd do `open ReactIntl;` at the top of the file.

Reply:
Done 👍